reconcile/gitlab_housekeeping.py

```diff
@@ -99,6 +120,8 @@ def handle_stale_items(dry_run, gl, days_interval, enable_closing, item_type):
     for item in items:
         item_iid = item.attributes.get("iid")
         item_labels = item.attributes.get("labels")
+        if AUTO_MERGE in item_labels and item.merge_status == "cannot_be_merged":
```

Comment:
This `cannot_be_merged` looks a bit magic but it actually comes from the API: https://docs.gitlab.com/ee/api/merge_requests.html#single-merge-request-response-notes

I would make it a constant and add the API reference link in a comment so that it looks less magical :)

Reply:
i'll do it in a follow up PR, since there are more occurences.

Reply:
#2681